internal/serialization/serialization.go

```diff
@@ -51,26 +51,36 @@ func NewService(serializationConfig *pubserialization.Config) (*Service, error)
 // ToData serializes an object to a Data.
 // It can safely be called with a Data. In that case, that instance is returned.
 // If it is called with nil, nil is returned.
-func (s *Service) ToData(object interface{}) (pubserialization.Data, error) {
-	if _, ok := object.(*SerializationData); ok {
-		return object.(*SerializationData), nil
+func (s *Service) ToData(object interface{}) (r *Data, err error) {
+	defer func() {
+		if rec := recover(); rec != nil {
+			err = hzerror.MakeError(rec)
+		}
+	}()
+	if serData, ok := object.(*Data); ok {
+		return serData, nil
 	}
-	dataOutput := NewPositionalObjectDataOutput(1, s, s.SerializationConfig.BigEndian)
+	dataOutput := NewPositionalObjectDataOutput(64, s, s.SerializationConfig.BigEndian)
```

Comment:
nit: why 64 bytes is used as the starting value? It would be nice to have a comment around this line to justify the choice. Node.js client uses `heap_data_header_size + long_data_size` (16 bytes) as the starting value.

Reply:
It's a bit arbitrary. Ideally the output size should be known before even creating the buffer, but I didn't yet implement that. I plan to do that in one of the subsequent previews.